src/main/java/io/github/filelize/Filelizer.java

```diff
@@ -23,7 +23,7 @@ public class Filelizer implements IFilelizer {
 
     public Filelizer(String basePath) {
         var objectMapper = new ObjectMapper();
-        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.findAndRegisterModules();
```

Comment:
Im not anough familiar with the findAndRegisterModules. Filelizer should solve the formatting in a best way without the user having to think of it

Reply:
This would do the same thing as finding `JavaTimeModule` if it's on the classpath (as it will be) and it will ALSO find modules for any other specific types that need to be supported; if a user type has a module associated with it, this would register those modules as well without requiring the user construct an `ObjectMapper` and pass it in. Thus, it would (IMO) fulfill the requirement better than specifically registering `JavaTimeModule`.